### PR TITLE
Simplify description of yellow LED

### PR DESCRIPTION
**Pull request by c0ffeeca7**

### CODE REVIEW

File: src/documentation/green-leds.html [hunk 1]
```diff
@@ -30,28 +30,8 @@ <h2>Activity LED (green)</h2>
...
-        <td>Off</td>
-        <td>System is in EEPROM firmware or boot loader mode.</td>
-      </tr>
-      <tr>
-        <td>Heartbeat</td>
-        <td>Home Assistant OS is running.</td>
-      </tr>
-    </tbody>
-  </table> 
+  <p>If the yellow LED blinks in a heartbeat pattern, the Home Assistant operating system is running.</p>
```
Comment (agners) [inline src/documentation/green-leds.html:33]:
```suggestion
  <p>The yellow system health LED blinks in a heartbeat pattern if the Home Assistant operating system is running.</p>
```

File: src/documentation/green-leds.html [hunk 2]
```diff
@@ -30,28 +30,8 @@ <h2>Activity LED (green)</h2>
...
-        <td>Off</td>
-        <td>System is in EEPROM firmware or boot loader mode.</td>
-      </tr>
-      <tr>
-        <td>Heartbeat</td>
-        <td>Home Assistant OS is running.</td>
-      </tr>
-    </tbody>
-  </table> 
+  <p>The yellow system health LED blinks in a heartbeat pattern if the Home Assistant operating system is running.</p>
```
Comment (frenck) [inline src/documentation/green-leds.html:33]: Except this is the docs for the Green?

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @

File: src/documentation/green-leds.html (hunk 2 above)
Comment (frenck) [reply src/documentation/green-leds.html:33]: OMG... 🤦 

ok, time to call it a day 🙈

**Review by frenck — APPROVED** [review]

**Pull request merged**